.gitignore

```diff
@@ -30,3 +31,6 @@
 *.exe
 *.out
 *.app
+
+# Vendor directory
+/vendor
```

Comment:
ditto,

Reply:
Without that rule, vendor directory appeared on the list of the unstaged changes, so I'm ignoring it explicitly.